public/js/app.js

```diff
@@ -0,0 +1,315 @@
+(function ($) {
+
+    $(document).ready(
+        function () {
+
+            /**************** Slider DEGRADE**********************/
+            var colors = new Array(
+[5, 4, 10],
+[33, 28, 66],
+[60, 51, 121],
+[88, 74, 176],
+[99, 86, 184],
+[238, 68, 55]);
+
+            var step = 0;
+            var colorIndices = [0, 1, 2, 3];
+
+            //transition speed
+            var gradientSpeed = 0.004;
+
+            function updateGradient() {
+
+                if ($ === undefined) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
                if ($ === undefined) {return;}
```